spanish/wg-diversity-inclusion/focus-areas/project-and-community/project-demographics.md

```diff
@@ -0,0 +1,63 @@
+- Ayudar a mantener un espacio seguro que considere a cada persona miembro perteneciente a comunidades diversas que colaboran en el proyecto, fomentando su participación.
+- Determinar si cada persona miembro de la comunidad es de origen diverso analizando los grupos de [junta de miembros](https://github.com/chaoss/wg-diversity-inclusion/blob/main/focus-areas/governance/board-council-diversity.md), mantenedores, organizadores de conferencias, y contribuyentes.
+- Determinar si las nuevas personas miembro de la comunidad son de orígenes diversos. 
+- Compartir información demográfica no sensible de la comunidad con otras personas.
+- Crear una declaración de confidencialidad de la información demográfica. 
+- Descubrir las historias menos visibles teniendo en cuenta la interseccionalidad de la información demográfica.
+- Identificar las áreas que permitan un potencial crecimiento en la inclusión, basado en los datos demográficos extraídos. 
+
+## Implementación 
+_El uso y diseminación de datos métricos de salud pueden violar las políticas de privacidad. Las organizaciones pueden estar expuestas a riesgos. Estos riesgos pueden provenir del cumplimiento del Reglamento General de Protección de Datos (RGPD) en la Unión Europea, o de la ley estatal en los Estados Unidos, o cualquier otra ley. Existe la posibilidad de riesgos contractuales derivados de las condiciones de servicio de proveedores de datos como GitHub y GitLab. El uso de las métricas debe ser examinado para detectar riesgos y posibles problemas de ética de los datos. Para obtener información adicional, ver el [Documento de Ética de los datos de CHAOSS]()._
+
+## Filtros 
+  - Datos demográficos como aquellos descritos en el repositorio de datos demográficos de CHAOSS. 
+
+  - Datos demográficos de distintos [tipos de contribuyentes](https://chaoss.community/metric-types-of-contributions/). Los tipos de contribución pueden incluir:
+    - Escribir código
+    - Revisar código 
+    - Detectar y corregir bugs 
+    - Control de calidad y testeo
+    - Actividades relacionadas a la seguridad informática
+    - Localización/L10N y traducción
+    - Organización de eventos
+    - Autoría del documento
+    - Construcción y gestión comunitaria
+    - Enseñar y crear tutoriales
+    - Solución de problemas y soporte técnico
+    - Trabajo creativo y diseño
+    - Accesibilidad, UI (Interfaz de Usuario), y UX (Experiencia de Usuario)
+    - Gestión de redes sociales 
+    - Atención al usuario y responder a preguntas que puedan surgir
+    - Escribir artículos 
+    - Relaciones Públicas - Entrevistas con la prensa técnica 
+    - Hablar en eventos
+    - Márketing y promoción de campañas 
+    - Desarrollo de sitios web
+    - Asesoría legal
+    - Gestión financiera
+
+## Estrategias de la recolección de datos 
+- Usa una encuesta para recopilar los datos demográficos del proyecto. Por ejemplo, puedes usar las preguntas Demográficas Abiertas. 
```

Comment:
Is there a link to the Demográficas Abiertas questions?

Reply:
no, there's not. Pls check the original file to check if something is missing :)